autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/AutofillManagementActivity.kt

```diff
@@ -96,6 +96,7 @@ class AutofillManagementActivity : DuckDuckGoActivity() {
                 } else {
                     finish()
                 }
+                viewModel.onAuthenticationEnded()
```

Comment:
At this point, the Auth activity has already been closed and onStart() should have been called.